CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# Contributing to autobrr
+
+Thanks for taking interest in contribution! We welcome anyone who wants to contribute.
+
+If you have an idea for a bigger feature or a change then we are happy to discuss it before you start working on it.
+It is usually a good idea to make sure it aligns with the project and is a good fit. Open an Issue or post in #dev-general on [Discord](https://discord.gg/WQ2eUycxyT).
+
+This document is a guide to help you through the process of contributing to autobrr.
+
+## Become a contributor
+
+* Code: new features, bug fixes, improvements
+* Report bugs
+* Documentation: The docs repo can be found here: [github.com/autobrr/autobrr.com](https://github.com/autobrr/autobrr.com)
+
+## Developer guide
+
+This guide helps you get started developing autobrr.
+
+## Dependencies
+
+Make sure you have the following dependencies installed before setting up your developer environment:
+
+- [Git](https://git-scm.com/)
+- [Go](https://golang.org/dl/) (see [go.mod](go.mod#L3) for minimum required version)
+- [Node.js](https://nodejs.org)
+- [pnpm](https://pnpm.io/installation)
+
+## How to contribute
+
+- **Fork and Clone:** [Fork the autobrr repository](https://github.com/autobrr/autobrr/fork) and clone it to start working on your changes.
+- **Branching:** Create a new branch for your changes. Use a descriptive name for easy understanding.
+  - Checkout a new branch for your fix or feature `git checkout -b fix/filters-issue`
+- **Coding:** Ensure your code is well-commented for clarity. With go use `go fmt`
+- **Commit Guidelines:** We appreciate the use of [Conventional Commit Guidelines](https://www.conventionalcommits.org/en/v1.0.0/#summary) when writing your commits.
+  - Examples: `fix(indexers): Mock improve parsing`, `feat(notifications): add NewService`
+  - There is no need for force pushing or rebasing. We squash commits on merge to keep the history clean and manageable.
+- **Pull Requests:** Submit a pull request from your Fork with a clear description of your changes. Reference any related issues.
+  - Mark it as Draft if it's still in progress.
+- **Code Review:** Be open to feedback during the code review process.
+
+## Development environment
+
+The backend is written in Go and the frontend is written in TypeScript using React.
+
+You need to have the Go toolchain installed and Node.js with `pnpm` as the package manager.
+
+Clone the project and change dir:
+
+```shell
+git clone github.com/YOURNAME/autobrr && cd autobrr
+```
+
+## Frontend
+
+First install the web dependencies:
+
+```shell
+cd web && pnpm install
+```
+
+Run the project:
+
+```shell
+pnpm start
```

Comment:
```suggestion
pnpm run dev
```

Reply:
I think simply `pnpm dev` is enough isn't it?

Reply:
Yes, pnpm dev is enough.
It assumes the run.